go.sum

```diff
@@ -212,6 +212,7 @@ github.com/stretchr/testify v1.4.0 h1:2E4SXV/wtOkTonXsotYi4li6zVWxYlZuYNCXe9XRJy
 github.com/stretchr/testify v1.4.0/go.mod h1:j7eGeouHqKxXV5pUuKE4zz7dFj8WfuZ+81PSLYec5m4=
 github.com/tektoncd/pipeline v0.9.0 h1:+CASYEwRr4SYpbi/lGpri7dvXSF1LdZcUo9xEQ2aQvk=
 github.com/tektoncd/pipeline v0.9.0/go.mod h1:IZzJdiX9EqEMuUcgdnElozdYYRh0/ZRC+NKMLj1K3Yw=
+github.com/tektoncd/pipeline v0.9.1 h1:nfhhQKxmo2TeL+LGR5pXlArEaFaRwnw7oAN3F0Gy55U=
```

Comment:
why this ?

Reply:
Rebased around 0.9.1 update, so this should be resolved.

Reply:
:+1: